build.sh

```diff
@@ -77,15 +77,19 @@ fi
 # Process flags
 if hasArg -v; then
     VERBOSE=1
+    NUMARGS=$((NUMARGS-1))
```

Comment:
I don't understand how this works. Why does decrementing NUMARGS cause it to build without installing?

Reply:
[dillon-cullinan 'scomment](https://github.com/rapidsai/cudf/pull/2981#pullrequestreview-327963286) answers this.

Anyway I removed this logic.